### DISCUSSION TITLE
Why is there no Chinese language setting？

### DISCUSSION THREAD

**Discussion by mutu555** [NationalSecurityAgency/ghidra#5202] [General]
I come from China and I really like this tool. It's really powerful and easy to use. But my English proficiency is very poor, which makes me love and hate this tool. I would like to have Chinese settings:)

**Comment by ryanmkurtz** [comment]
We do not have the resources to support this.

**Reply by mutu555** [reply]
Sorry, I didn't take this into account. But I am very grateful to you for creating this great software.
